docs/tools/sharding.md

```diff
@@ -0,0 +1,48 @@
+[**UDMI**](../../) / [**Docs**](../) / [**Tools**](./) / [Sharding](#)
+
+# Sharding Sequencer
+
+It's possible to run sequencer on an alternate set of project registries, called shards. This is
```

Comment:
If I'm understanding correctly there is one alternative registry/shard. It would be great if we had many shards, not just one, as long as we are here. What do you think?

Reply:
Yes (current setup), and yes (have many). It's why I called everything a "shard" rather than just suffix or something. But, I don't have the capacity to do that right now (it's not trivial) so not something I can prioritize and work into this PR.